client/commands.c

```diff
@@ -854,34 +880,38 @@ int xdag_show_state(xdag_hash_t hash)
-		"  run                 - run node after loading local blocks if option -r is used\n"
-		"  state               - print the program state\n"
-		"  stats               - print statistics for loaded and all known blocks\n"
-		"  terminate           - terminate both daemon and this program\n"
-		"  xfer S A            - transfer S our %s to the address A\n"
-		"  disconnect O [A|IP] - disconnect all connections or specified miners\n"
-		"                         O is option, can be all, address or ip\n"
-		"                         A is the miners' address\n"
-		"                         IP is the miners' IP\n"
+		"  account [N]          - print first N (20 by default) our addresses with their amounts\n"
+		"  balance [A]          - print balance of the address A or total balance for all our addresses\n"
+		"  block [A]            - print extended info for the block corresponding to the address or hash A\n"
+		"  lastblocks [N]       - print latest N (20 by default, max limit 100) main blocks\n"
+		"  exit                 - exit this program (not the daemon)\n"
+		"  help                 - print this help\n"
+		"  keygen               - generate new private/public key pair and set it by default\n"
+		"  level [N]            - print level of logging or set it to N (0 - nothing, ..., 9 - all)\n"
+		"  miners               - for pool, print list of recent connected miners\n"
+		"  mining [N]           - print number of mining threads or set it to N\n"
+		"  net command          - run transport layer command, try 'net help'\n"
+		"  pool [CFG]           - print or set pool config; CFG is miners:maxip:maxconn:fee:reward:direct:fund\n"
+		"                          miners - maximum allowed number of miners,\n"
+		"                          maxip - maximum allowed number of miners connected from single ip,\n"
+		"                          maxconn - maximum allowed number of miners with the same address,\n"
+		"                          fee - pool fee in percent,\n"
+		"                          reward - reward to miner who got a block in percent,\n"
+		"                          direct - reward to miners participated in earned block in percent,\n"
+		"                          fund - community fund fee in percent\n"
+		"  run                  - run node after loading local blocks if option -r is used\n"
+		"  state                - print the program state\n"
+		"  stats                - print statistics for loaded and all known blocks\n"
+		"  terminate            - terminate both daemon and this program\n"
+		"  xfer S A             - transfer S our %s to the address A\n"
+		"  disconnect O [A|IP]  - disconnect all connections or specified miners\n"
+		"                          O is option, can be all, address or ip\n"
+		"                          A is the miners' address\n"
+		"                          IP is the miners' IP\n"
+		"  white [-a|-l|-d][IP] - http rpc white list manager\n"
```

Comment:
I would suggest `rpc_white` command. `white` can create additional confusion since we already have whitelist for pools.